src/zenml/client.py

```diff
@@ -1227,8 +1227,8 @@ def list_stacks(
         size: int = PAGE_SIZE_DEFAULT,
         logical_operator: LogicalOperators = LogicalOperators.AND,
         id: Optional[Union[UUID, str]] = None,
-        created: Optional[datetime] = None,
-        updated: Optional[datetime] = None,
+        created: Optional[Union[datetime, str]] = None,
```

Comment:
Not really sure if the `str` would work here. As far as I can see, in the `aws_stack_deployment`, you are passing a value like `"gt:{str(date_start.replace(microsecond=0))}"` to the parameter `created` if a `date_start` is provided. This value is then used to create a stack filter model and in filter models, the definition of `datetime` filters is as follows: 


```python
    @staticmethod
    def _define_datetime_filter(
        column: str, value: Any, operator: GenericFilterOps
    ) -> NumericFilter:
        """Define a datetime filter for a given column.

        Args:
            column: The column to filter on.
            value: The datetime value by which to filter.
            operator: The operator to use for filtering.

        Returns:
            A Filter object.

        Raises:
            ValueError: If the value is not a valid datetime.
        """
        try:
            if isinstance(value, datetime):
                datetime_value = value
            else:
                datetime_value = datetime.strptime(
                    value, FILTERING_DATETIME_FORMAT
                )
        except ValueError as e:
            raise ValueError(
                "The datetime filter only works with values in the following "
                f"format: {FILTERING_DATETIME_FORMAT}"
            ) from e
        datetime_filter = NumericFilter(
            operation=GenericFilterOps(operator),
            column=column,
            value=datetime_value,
        )
        return datetime_filter```
        
and the `FILTERING_DATETIME_FORMAT` is `"%Y-%m-%d %H:%M:%S"`. The fact that the value starts with `gt:` might mess things up here.

Reply:
That `gt:` is already being parsed ahead of this method as a generic implementation for all our filter models.

Reply:
I can confirm I tested this and it works as expected